danger-klaxit/lib/danger_plugin.rb

```diff
@@ -56,9 +56,11 @@ def fail_for_bad_commits
 
   # Start a scan using `danger-brakeman_scanner`. Arguments are deduced from
   # the current path, and klaxit naming conventions (i.e `klaxit-<name>`).
-  def run_brakeman_scanner
-    configs = { app_path: "." }
-    configs.merge!(github_repo: github_repo.to_s) if github_repo
+  def run_brakeman_scanner(configs = { app_path: "." })
+    configs[:github_repo] = github_repo.to_s if github_repo
```

Comment:
```suggestion
    configs[:github_repo] ||= github_repo.to_s if github_repo
```
just saw this one :/

Reply:
I write it like this on purpose, because : 

- `:github_repo` is meant to be guessed internally
- the previous code was overwriting the setting

Do you see a case where we want to force the setting on `run_brakeman_scanner ` call?

Reply:
The previous code did not take settings into account at all. It was defining its own. Now that we let the user give us a `configs` hash. He should expect that the configurations he wrote on purpose are not overridden IMHO. And this won't affect any of our existing codebase since if configs is left empty, the default hash doesn't contain  `:github_repo`. But whatever, this is no big deal anyway !